shared/lib/pin_manager.py

```diff
@@ -216,6 +216,18 @@ def create_i2c(self, scl, sda, frequency=100000):
             (lambda: busio.I2C(scl, sda, frequency=frequency)),
         )
 
+    def create_uart(self, tx, rx, baudrate=50000):
+        """
+        Creates and returns ManagedDevice wrapping a busio.UART on the two specified pins
+        with the specified baudrate, or returns one from the cache if one has already
+        been created.
+        """
+        return self._create_general_device(
+            [tx, rx],
+            (busio.UART, baudrate),
+            (lambda: busio.UART(tx, rx, baudrate=baudrate)),
```

Comment:
There's no need to explicitly set the baudrate in the constructor for a `busio.UART`, since it can be reset through the `baudrate` property at a later time. It's alright to include it, but it's just not needed.

It would make more sense to include the `bits`, `parity`, and `stop` parameters since some devices require different values for them and they can't be reconfigured later. I don't think we're interested in hardware RTS/CTS for RS232 or hardware TE for RS485, so those parameters are less necessary, but again they can't be reconfigured after construction, so they're candidates for inclusion here.